cSploit/src/org/csploit/android/MainActivity.java

```diff
@@ -139,15 +145,16 @@ private void createUpdateLayout() {
         lv.setVisibility(View.GONE);
         findViewById(R.id.textView).setVisibility(View.GONE);
 
-        createUpdateStatusText();
-
-        mUpdateStatus
-                .setText(UPDATE_MESSAGE.replace("#STATUS#", "..."));
+        mUpdateStatus.setVisibility(View.VISIBLE);
```

Comment:
calling a method of `mUpdateStatus`, but checking if it is null after few lines of code.
if all path that lead the program here initializes `mUpdateStatus` the null check is pointless, instead if some execution path leave it uninitialized a NullPointerException will be raised.